### PR TITLE
[bugfix] Autoexpand is not working properly due to height fix

### PR DESCRIPTION
**Pull request by besuhoff**
- Add min-height parameter that sets minimal input height in pixels

### CODE REVIEW

File: src/javascript/app/ng-wig/ngWigEditableDirective.js [hunk 1]
```diff
@@ -46,7 +53,7 @@ angular.module('ngWig').directive('ngWigEditable', function () {
           if (!scope.autoexpand) {
             var height = scope.originalHeight;
           } else {
-            height = angular.element($document.documentElement).outerHeight();
+            height = angular.element($document.documentElement).outerHeight() + 1;
```
Comment (stevermeister) [inline src/javascript/app/ng-wig/ngWigEditableDirective.js]: why "+1"? :)

File: src/index.html [hunk 2]
```diff
@@ -16,7 +16,7 @@
 <div class="container">
 
     <h1>One simple WYSIWYG</h1>
-    <textarea ng-wig="text1"></textarea>
+    <textarea ng-wig="text1" min-height="200"></textarea>
```
Comment (stevermeister) [inline src/index.html]: could we do it without extra attribute?

Comment (besuhoff) [inline src/index.html]: How do we set the height otherwise?

File: src/javascript/app/ng-wig/ngWigEditableDirective.js (hunk 1 above)
Comment (besuhoff) [inline src/javascript/app/ng-wig/ngWigEditableDirective.js]: Hell knows. Probably a border, but not sure

File: src/index.html (hunk 2 above)
Comment (stevermeister) [inline src/index.html]: with styles?

File: src/javascript/app/ng-wig/ngWigEditableDirective.js (hunk 1 above)
Comment (stevermeister) [inline src/javascript/app/ng-wig/ngWigEditableDirective.js]: maybe outerHeight does not work correctly, or you just copied it?

File: src/index.html (hunk 2 above)
Comment (besuhoff) [inline src/index.html]: with styles of what?

Comment (stevermeister) [inline src/index.html]: of height, no?

File: src/javascript/app/ng-wig/ngWigEditableDirective.js (hunk 1 above)
Comment (besuhoff) [inline src/javascript/app/ng-wig/ngWigEditableDirective.js]: you're right, it's just some magic number. Removed.

File: src/index.html (hunk 2 above)
Comment (besuhoff) [inline src/index.html]: of height of what?

**Comment by stevermeister** [comment]
we decided to do it another way, by removing iframe

**Pull request closed without merging**